File: af.rb
#!/usr/bin/env ruby

require "cytoplasm/afshell"

if ARGV.length.to_i==0
    puts " "
    puts "             ,,,,,,,,"
    puts "           ,|||````||||"
    puts "     ,,,,|||||       ||,"
    puts "  ,||||```````       `||"
    puts ",|||`                 |||,"
    puts "||`     ....,          `|||"
    puts "||     ::::::::          |||,"
    puts "||     :::::::'     ||    ``|||,"
    puts "||,     :::::'               `|||"
    puts "`||,                           |||"
    puts " `|||,       ||          ||    ,||"
    puts "   `||                        |||`"
    puts "    ||                   ,,,||||"
    puts "    ||              ,||||||```"
    puts "   ,||         ,,|||||`"
    puts "  ,||`   ||   |||`"
    puts " |||`         ||"
    puts ",||           ||"
    puts "||`           ||"
    puts "|||,         |||"
    puts " `|||,,    ,|||"
    puts "   ``||||||||`"
    puts " "
    puts " "

  while true
    printf("AFshell > ")
    STDOUT.flush
    input=gets.chomp
    command, parameter=input.split(" ")
  
    current_afShell=AFshell.new(command, parameter)
    current_afShell.execute_command()
  end
  
else
  #Speed launcher
end

File: af.py
#!/usr/bin/env python

import sys

from cytoplasm.af_shell import AFshell

def main():
    counter=0
    
    for item in sys.argv:
        counter+=1
        
    if counter==1:
        
        print " "
        print "             ,,,,,,,,"
        print "           ,|||````||||"
        print "     ,,,,|||||       ||,"
        print "  ,||||```````       `||"
        print ",|||`                 |||,"
        print "||`     ....,          `|||"
        print "||     ::::::::          |||,"
        print "||     :::::::'     ||    ``|||,"
        print "||,     :::::'               `|||"
        print "`||,                           |||"
        print " `|||,       ||          ||    ,||"
        print "   `||                        |||`"
        print "    ||                   ,,,||||"
        print "    ||              ,||||||```"
        print "   ,||         ,,|||||`"
        print "  ,||`   ||   |||`"
        print " |||`         ||"
        print ",||           ||"
        print "||`           ||"
        print "|||,         |||"
        print " `|||,,    ,|||"
        print "   ``||||||||`"
        print " "
        print " "
        
        while 1:
            sys.stdout.flush()
            user_input=raw_input("AFshell> ")
            if user_input.find(" ")>=0:
                command, parameter=user_input.split(" ")
                current_AFshell=AFshell(command, parameter)
                current_AFshell.execute_command()
            else:
                command=user_input
                current_AFshell=AFshell(command, None)
                current_AFshell.execute_command()

if __name__ == "__main__":
    main()